delivery-slots.js

```diff
@@ -110,4 +113,13 @@ async function run() {
   }
 }
 
-run();
+if (config.cron) {
+  schedule.scheduleJob(config.cron, () => run());
```

Comment:
Can we still keep a log when it runs on the cron?

Reply:
this is an outdated version, the new version logs the execution time for both modes.

Reply:
Do you need to do a new pull request to get the newer version commit in?

Reply:
no, you should see the latest version in the files tab, specifically this is where the run time log is at now https://github.com/paulmaunders/delivery-slot-bot/pull/2/files#diff-82828ea2d34b83ab67542f306b66de78R24-R27